ts/services/addGlobalKeyboardShortcuts.ts

```diff
@@ -55,10 +55,7 @@ export function addGlobalKeyboardShortcuts(): void {
       const focusedIndexes: Array<number> = [];
 
       targets.forEach((target, index) => {
-        if (
-          (focusedElement != null && target === focusedElement) ||
-          target.contains(focusedElement)
-        ) {
+        if (target.contains(focusedElement)) {
```

Comment:
`target.contains(focusedElement)` makes the preceding condition pointless because [`.contains()`](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains) allows a `null` parameter and considers elements to also contain themselves.